edgelet/edgelet-docker/src/migc_persistence.rs

```diff
@@ -65,13 +65,13 @@ impl MIGCPersistence {
         // read MIGC persistence file into in-mem map
         // this map now contains all images deployed to the device (through an IoT Edge deployment)
         let image_map = match get_images_with_timestamp(guard.filename.clone()) {
-                Ok(map) => map,
-                Err(e) => {
-                    drop(guard);
-                    log::error!("Could not read auto-prune data; image garbage collection did not prune any images: {}", e);
-                    return HashMap::new();
-                },
-            };
+            Ok(map) => map,
+            Err(e) => {
+                drop(guard);
+                log::error!("Could not read auto-prune data; image garbage collection did not prune any images: {}", e);
+                return HashMap::new();
```

Comment:
Here did we want to create file if it doesn't exist?

Reply:
Currently my understanding is that we recreate the file because we are creating an empty map